
# BUGS

Note the python scripts doesn't run as person aythenticated as `gcloud auth login` but rather with ADC.
